### PR TITLE
Implemented Meeting & Team Editor Sections

### LINKED ISSUE #58: Flatten site JSX
__Description:__
Remove content of <Content /> and place in <Site /> component

__Requires:__
* <Site /> component to be set up
* Child components to place in <Site /> (e.g. meeting, members/team)

__Breakdown:__
- [x] Move components up from <Content /> to <Site />
- [x] Remove <Content /> component
- [x] Check that props are handed down properly

__Time Estimation:__
1 hrs.

### PR DESCRIPTION
**Pull request by okjuan**
Only using text input fields for now. May want to change to a 'select' tag/component in the future, for example. Resolves #58 #63

### CODE REVIEW

File: client/components/pages/editor/sections/Meeting.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import LongTextField from 'parts/LongTextField'
+import ShortTextField from 'parts/ShortTextField'
+
+export default class Meeting extends Component{
+  constructor(props){
+    super(props)
+  }
+
+  static Title = 'Meeting Info'
```
Comment (bndnio) [inline client/components/pages/editor/sections/Meeting.jsx:11]: Do we need this? global-type variables in react is an anti-patern

File: client/components/pages/editor/sections/Team.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React, {Component, PropTypes} from 'react'
+
+import ClubRepField from 'parts/ClubRepField'
+
+export default class Team extends Component{
+  constructor(props){
+    super(props)
+  }
+
+  static Title = 'Team'
```
Comment (bndnio) [inline client/components/pages/editor/sections/Team.jsx:10]: This too?

File: client/components/pages/editor/sections/Meeting.jsx (hunk 1 above)
Comment (hwkr) [reply client/components/pages/editor/sections/Meeting.jsx:11]: That's how the label is set in the editor section

File: client/components/pages/editor/sections/Team.jsx (hunk 2 above)
Comment (hwkr) [reply client/components/pages/editor/sections/Team.jsx:10]: Same reason

File: client/components/pages/editor/sections/Hero.jsx [hunk 3]
```diff
@@ -19,13 +19,8 @@ export default class Hero extends Component{
...
   handleChange = (field, value, index) => {
     const d = this.props.data
-
-    if (index != null){
-      d[field][index] = value
-
-    } else {
-      d[field] = value
-    }
+    if (index === undefined || index === null) d[field] = value
```
Comment (hwkr) [inline client/components/pages/editor/sections/Hero.jsx:22]: Fixed now!

Comment (okjuan) [reply client/components/pages/editor/sections/Hero.jsx:22]: tbh not a fan of leaving out the curly braces -- lends itself to bugs when new code is added

**Pull request merged**